### PR TITLE
Add fame.is-a-dev

### PR DESCRIPTION
**Pull request by TheRealFame**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [✓ ] The file is in the `domains` folder and is in the JSON format.
- [✓ ] The file's name is all lowercased and alphanumeric. 
- [ ✓] You have completed your website. 
- [ ✓] The website is reachable.  
- [ ✓] You're not using Vercel.  
- [ ✓] The CNAME record doesn't contain `https://` or `/`.  
- [ ✓] There is sufficient information at the `owner` field.  

## Website Link/Preview

[cutefame.net](https://cutefame.net)

### CODE REVIEW

File: domains/fame.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "description": "Redirection to the games side of my site.",
+    "Current site i want it to redirect to": "https://cutefame.net/games",
+    "owner": {
+        "username": "<TheRealFame>",
```
Comment (Stef-00012) [inline domains/fame.json:5]: Remove <>
```suggestion
        "username": "TheRealFame",
```

File: domains/fame.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+        "email": "email@cutefame.net",
+        "twitter": "<The_RealFame>"
```
Comment (Stef-00012) [inline domains/fame.json:7]: Remove <>
```suggestion
        "twitter": "The_RealFame"
```

File: domains/fame.json
Comment (Stef-00012) [inline domains/fame.json:1]: Invalid JSON
Missing records

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/fame.json
Comment (TheRealFame) [reply domains/fame.json:1]: Records?

**Comment by TheRealFame** [comment]
The requested cname for an .is-a-dev domain should be there now, I just don't know if my current json on my branch is invalid, or the one i made through documentation is.

**Comment by TheRealFame** [comment]
@Stef-00012

**Pull request closed without merging**